draft-ietf-rats-concise-ta-stores.md

```diff
@@ -74,9 +74,9 @@ Trust anchor (TA) stores may be used for several purposes in the Remote Attestat
 
 The RATS architecture {{RFC9334}} uses the definition of a trust anchor from {{RFC6024}}: "A trust anchor represents an authoritative entity via a public key and associated data.  The public key is used to verify digital signatures, and the associated data is used to constrain the types of information for which the trust anchor is authoritative." In the context of RATS, a trust anchor may be a public key or a symmetric key. This document focuses on trust anchors that are represented as public keys.
 
-The Concise Reference Integrity Manifest (CoRIM) {{I-D.draft-ietf-rats-corim}} specification defines a binary encoding for reference values using the Concise Binary Object Representation (CBOR) {{RFC8949}}. Amongst other information, a CoRIM may include key material for use in verifying evidence from an attesting environment (see section 3.11 in {{I-D.draft-ietf-rats-corim}}). The extension in this document aims to enable public key material to be decoupled from reference data for several reasons, described below.
+The Concise Reference Integrity Manifest (CoRIM) {{I-D.draft-ietf-rats-corim}} specification defines a binary encoding for reference values and endorsmenents using the Concise Binary Object Representation (CBOR) {{RFC8949}}. Amongst other information, a CoRIM may include key material for use in verifying evidence from an attesting environment (see {{Section 3.1.4.4 and 3.1.4.5 of I-D.draft-ietf-rats-corim}}). The extension in this document aims to enable public key material to be decoupled from reference data and other kinds of endorsement data for several reasons, described below.
```

Comment:
```suggestion
The Concise Reference Integrity Manifest (CoRIM) {{I-D.draft-ietf-rats-corim}} specification defines a binary encoding for Reference Values and Endorsements using the Concise Binary Object Representation (CBOR) {{RFC8949}}. Amongst other information, a CoRIM may include key material for use in verifying Evidence from an attesting environment (see {{Section 3.1.4.4 and 3.1.4.5 of I-D.draft-ietf-rats-corim}}). The extension in this document aims to enable public key material to be decoupled from reference data and other kinds of endorsement data for several reasons, described below.
```

Reply:
I prefer we do this kind of changes in a separate PR which sweeps the entire document rather than just the intro.